solutions/slice-matrix

```diff
@@ -0,0 +1,36 @@
+let fs = require("fs");
+let data = fs.readFileSync(0, 'utf-8');
+let idx = 0;
+data = data.split('\n');
+
+function readLine() {
+  idx++;
+  return data[idx - 1];
+}
+
+// -------- Do NOT edit anything above this line ----------
+// Use readLine() for taking input, it will read one line of from the input  and is stored in string format
+let rowColumn = readLine().split(" ").map(Number);
+let rows=rowColumn[0];
+let columns=rowColumn[1];
+let array=[];
+for(let row=0;row<rows;row++)
+{
+	let	temporaryArray = readLine().split(" ").map(Number);
+	array.push(temporaryArray);
+}
+let submatrix = readLine().split(" ").map(Number);
```

Comment:
@DevratAgarwal  use the variable name `submatrix` as `margins`.

Reply:
@shivateja1729  done